config.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+class Config(object):
+    SECRET_KEY = os.environ.get('SECRET_KEY') or 'you-will-never-guess'
```

Comment:
os.environ.get принимает вторым аргументом значение по умолчанию
```suggestion
    os.environ.get('SECRET_KEY', 'you-will-never-guess')
```